database/postgres/data.js

```diff
@@ -0,0 +1,139 @@
+const faker = require('faker');
+const md5 = require('md5');
+
+const DESCRIPTION = 'description';
+const INFO = 'info';
+const SIMILARITEMS = 'similarItems';
+const CONFIGURATION = 'configuration';
+
+// configure objects for bulk insert
+/**
+ 
+  {
+    create: create table query
+    drop: drop table query
+    insert: insertion query
+    record: function to generate record. passed index and total args. return string with quoted values, seperated by commas
+    postInsert: run after all records are inserted(key constraints etc)
+    preInsert: run after table made before records inserted
+
+  }
+ */
```

Comment:
this pseudocode is helpful to understand what is happening below. thanks for including it.